app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @age_sorted_actors = @movie.actors.sort_by_age
+  end
```

Comment:
`@movie.actors.sort_by_age` is also ok to do in the view since you are calling on an instance method to do the work and not doing the work directly in the view with AR.